app/config/services.yml

```diff
@@ -4,6 +4,5 @@ parameters:
 #    parameter_name: value
 
 services:
-#    service_name:
-#        class: AppBundle\Directory\ClassName
+#    'My\ClassName':
```

Comment:
One question: in your tests, how can you replace this service with another one (to use stubs for instance)?